editor/editor_themes.cpp

```diff
@@ -852,7 +852,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 
 	Ref<StyleBoxFlat> style_tree_cursor = style_default->duplicate();
 	style_tree_cursor->set_draw_center(false);
-	style_tree_cursor->set_border_width_all(border_width);
+	style_tree_cursor->set_border_width_all(MAX(EDSCALE, border_width));
```

Comment:
I'm not sure if `MAX(EDSCALE, ...)` is the right approach here. Otherwise, the border will be forced to be two pixels wide as soon as the editor scale is greater than 100% (e.g. 125%). This is because floats are always rounded up when casted to integers in C++.

```suggestion
	style_tree_cursor->set_border_width_all(MAX(1, border_width));
```

Reply:
> This is because floats are always rounded up when they're cast to integers in C++.

That's not correct, floats are truncated when cast to `int` (basically keeping only the integral part), not rounded.

```
#include <iostream>

int main() {
  std::cout << (int)1.25 << std::endl;
  std::cout << (int)-1.25 << std::endl;
  return 0;
}
```
prints
```
1
-1
```